microsoft-edge/progressive-web-apps-chromium/whats-new/pwa.md

```diff
@@ -13,6 +13,39 @@ This page lists the updates to PWAs and Web Apps from the Microsoft Edge Web App
 
 
 <!-- ====================================================================== -->
+## What's New in Microsoft Edge 100
+Microsoft Edge Canary/Dev channels moved to version 100 on February 9,2022 and in this milestone, we are testing the following new web app features. 
+
+### App Sync
+The sync service powering features like favorites, settings, history & more has been expanded to include your installed web apps - that is PWAs and sites installed as apps. The sync service works when you are signed in to your profile and have opted in to syncing. As we test and gradually release this feature over the coming weeks, users that have opted in to sync will start to see "Available apps" in the `edge://apps` page and the [new Apps menu](#redesigned-apps-menu). 
+
+Available apps will show any apps you've installed on other devices with the same signed in profile, that aren't currently installed in the Edge instance you are using. The available apps will be shown in their own section with grey icons. Press the "Install" button to install the app on that device.
+
+   :::image type="content" source="media/available-app-install.png" alt-text="Available apps shown on Apps page with install button highlighted.":::
```

Comment:
Note: there is no need to indent these images. Also, @mikehoffms can confirm, but we've been moving towards using normal markdown image tagging (i.e. `![alt text](/path/to/img/png)`) instead of the `:::image:::` extension.

Reply:
I'm fixing as part of routine Writer/Editor pass